openquake/hazardlib/contexts.py

```diff
@@ -208,8 +215,6 @@ def poe_map(self, src, sites, imtls, trunclevel, rup_indep=True):
             initvalue=rup_indep)
         eff_ruptures = 0
         with self.ir_mon:
```

Comment:
I thought that setting `src.magnitude_scaling_relationship = pointMSR` was needed

Reply:
Why did you think that? I don't see why it would be needed, since we are just replacing rjb and repi (which have been computed using any MSR) with the equivalent distances.

Maybe Marco can comment in case I'm misunderstanding, since I think he discussed with you the previous PR.